priv/proto3/google/protobuf/api.proto

```diff
@@ -172,7 +172,7 @@ message Method {
 // The mixin construct implies that all methods in `AccessControl` are
 // also declared with same name and request/response types in
 // `Storage`. A documentation generator or annotation processor will
-// see the effective `Storage.GetAcl` method after inherting
+// see the effective `Storage.GetAcl` method after inheriting
```

Comment:
I'd actually like this typo to not be corrected, since it is in a vendored file.

Reply:
> 

Done.Reverted.